cfbs/commands.py

```diff
@@ -240,24 +240,26 @@ def init_command(index=None, masterfiles=None, non_interactive=False) -> int:
     CFBSConfig.reload()
 
     branch = None
-    to_add = ""
+    to_add = []
     if masterfiles is None:
         if prompt_user(
             non_interactive,
             "Do you wish to build on top of the default policy set, masterfiles? (Recommended)",
             choices=YES_NO_CHOICES,
             default="yes",
         ) in ("yes", "y"):
-            to_add = "masterfiles"
+            to_add = ["masterfiles"]
         else:
-            to_add = prompt_user(
-                non_interactive,
-                "Specify policy set to use instead (empty to skip)",
-                default="",
-            )
+            to_add = [
+                prompt_user(
+                    non_interactive,
+                    "Specify policy set to use instead (empty to skip)",
+                    default="",
```

Comment:
Perhaps an example response. I guess, but am not sure that I should provide a module repo URL and perhaps a module name, maybe a commit sha in the URL?